SPECS/rubygem-yajl-ruby/rubygem-yajl-ruby.spec

```diff
@@ -0,0 +1,35 @@
+%global debug_package %{nil}
+%global gemdir %(IFS=: R=($(gem env gempath)); echo ${R[${#R[@]}-1]})
+%global gem_name yajl-ruby
+Summary:        A streaming JSON parsing and encoding library for Ruby
+Name:           rubygem-yajl-ruby
+Version:        1.4.1
+Release:        1%{?dist}
+License:        MIT
+Vendor:         Microsoft Corporation
+Distribution:   Mariner
+Group:          Development/Languages
+URL:            https://rubygems.org/gems/%{gem_name}/versions/%{version}
+Source0:        https://rubygems.org/downloads/%{gem_name}-%{version}.gem
+BuildRequires:  ruby >= 1.8.6
+
+%description
+a C binding to the excellent YAJL JSON parsing and generation library.
```

Comment:
>excellent [](start = 19, length = 9)

I know we didn't write this, but can we avoid making endorsement claims of this nature?

Reply:
Resolved